prov/efa/test/efa_unit_test_info.c

```diff
@@ -151,3 +152,75 @@ void test_info_check_shm_info()
 	test_info_check_shm_info_from_hints(hints);
 
 }
+
+/**
+ * @brief Check that cases when a user requested FI_HMEM support
+ *        using different version of libfabric API, the EFA provider
+ *        behavior as expected on a system has CUDA
+ */
+void test_info_check_hmem_cuda_support_on_api_version()
+{
+	struct fi_info *hints, *info = NULL;
+	int err;
+
+	if (!hmem_ops[FI_HMEM_CUDA].initialized)
+		return;
```

Comment:
Instead of return(which means the test succeeded without testing anything), we should skip.

Reply:
used skip in new version